src/styles/blocks/card.scss

```diff
@@ -0,0 +1,100 @@
+body {
+  margin: 0;
+  font-family: $main-font;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.card {
+  padding: 16px;
+  padding-top: 32px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: $main-white-color;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.card__image {
+  box-sizing: border-box;
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.card__image img {
+  width: 100%;
+  height: 100%;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 0;
+  margin-bottom: 4px;
+}
+
+.card__code {
+  align-self: flex-start;
+  color: $main-color-text;
+  margin: 0;
+  margin-bottom: 16px;
+}
+
+.card__assessment {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 24px;
+}
+
+.card__price {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.card__price-text {
+  font-size: 12px;
+  color: $main-color-text;
+}
+
+.card__price-cost {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__button {
+  box-sizing: border-box;
+  width: 100%;
+  height: 40px;
+  font-family: $main-font;
+  color: $main-white-color;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  background-color: $color-batton;
+  border-radius: 5px;
+  border: 1px solid $color-batton;
```

Comment:
Ensure that the variable `$color-batton` is defined. If it's not defined, it will cause a compilation error.